CONTRIBUTING.md

```diff
@@ -1,17 +1,39 @@
-# Contributing to the community repo
+# Contributing to the Community Repo
 
-Make a [pull request](https://help.github.com/articles/using-pull-requests) (PR).
+Welcome to the Kubernetes Community contributing guide. We are excited about the prospect of you joining our [community](https://github.com/kubernetes/community]!
 
-Upon successful review, someone will give the PR
-a __LGTM__ (_looks good to me_) in the review thread.
+## Finding Something to Work On
+
+Before you begin, you can look at some places where you can help out:
+
+- [Existing open issues](https://github.com/kubernetes/community/issues)
```

Comment:
Should we point to this repo only?

Reply:
Since this file is at the root of this repo I think this makes sense. We should probably revise k/k/contributing.md also.

Reply:
@castrojo is the goal of the document - documenting the contribution guidelines to kubernetes/community only, or to entire kubernetes codebase (everything under Kubernetes organization on GitHub)?

Reply:
@idvoretskyi This should be for k/community, however, k/k basically points to this repo and the developer guide. I think that step 2 should be revising k/k/contributing.md to point to the new developer guide (once complete).